packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -72,8 +73,10 @@ export class ProfilesCache {
         this.cwd = cwd != null ? getFullPath(cwd) : undefined;
     }
 
-    public async getProfileInfo(): Promise<zowe.imperative.ProfileInfo> {
-        const mProfileInfo = new zowe.imperative.ProfileInfo("zowe");
+    public async getProfileInfo(envTheia = false): Promise<zowe.imperative.ProfileInfo> {
+        const mProfileInfo = new zowe.imperative.ProfileInfo("zowe", {
+            credMgrOverride: zowe.imperative.ProfileCredentials.defaultCredMgrWithKeytar(() => getSecurityModules("keytar", envTheia)),
```

Comment:
Anyone know why codecov seems so angry that this line isn't covered by tests? This object is being passed into Imperative, I feel like there isn't a way for this line to not be covered when the rest are.

Reply:
from what I can tell, it might be that the arrow function inside the function call `zowe.imperative.ProfileCredentials.defaultCredMgrWithKeytar` is not being tested

Reply:
Well, I am testing the defaultCredMgrWithKeytar function now. So the only other thing it could be is getSecurityModules, but I don't know how to spy on that since it is a direct import, not a class and function.